packages/react-instantsearch/src/components/CurrentRefinements.js

```diff
@@ -13,8 +13,21 @@ class CurrentRefinements extends Component {
     })).isRequired,
     refine: PropTypes.func.isRequired,
     transformItems: PropTypes.func,
+    isEmpty: PropTypes.func,
```

Comment:
Did you think about other namings for this? Empty is only relevant to list of items I believe, maybe "canRefine"? canRefine true means that the widget can be used to refine, false it means it cannot be used to refine.